src/khoj/interface/web/chat.html

```diff
@@ -335,7 +348,14 @@
             box-shadow: 0 0 11px #aaa;
         }
         #chat-input {
-            font-size: medium;
+            font-family: roboto, karma, segoe ui, sans-serif;
+            font-size: 18px;
```

Comment:
`font-size` is already set to medium. Is there a specific reason we want to set font-size to 18px? If not, it may better to let it be the same as before (i.e `font-size: medium`)

Reply:
There is no special reason, just for aesthetics.

Reply:
I'd say let's revert it to `font-size: medium`. This makes it a lot easier to maintain css when you don't have explicitly defined settings in a bunch of small places to track.